packages/themes/blue-jeans/src/spacing/spacing-concepts.scss

```diff
@@ -27,49 +26,51 @@ $inline-concept: (
         "07": map-get($sizes, "07"), // 20px
         "08": map-get($sizes, "08")  // 24px
 );
-
 $inset-concept: $inline-concept;
 $squish-concept: $inset-concept;
 $stretch-concept: $inset-concept;
-
 $border-radius-concept: (
         "01": map-get($sizes, "03"), // 4px
         "02": map-get($sizes, "04"), // 8px
         "03": map-get($sizes, "06")  // 16px
 );
 
+/* stylelint-disable function-calc-no-unspaced-operator */
```

Comment:
why?

Reply:
for some reason, the kebal-case variable name misread by linter and counts as substraction